homeassistant/components/romy/config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Config flow for ROMY integration."""
+from __future__ import annotations
+
+import romy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import zeroconf
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, LOGGER
+
+
+def _schema_with_defaults(
+    host: str = "", name: str = "", requires_password: bool = False
+) -> vol.Schema:
+    schema = {
+        vol.Required(CONF_HOST, default=host): cv.string,
+        vol.Optional(CONF_NAME, default=name): cv.string,
+    }
```

Comment:
```suggestion
    default_values: dict[str, Any] = {}, requires_password: bool = False
) -> vol.Schema:
    schema = {
        vol.Required(CONF_HOST, default=default_values.get(CONF_HOST, "")): cv.string,
        vol.Optional(CONF_NAME, default=default_values.get(CONF_NAME,"")): cv.string,
    }
```

Reply:
adjust and set it to None, empty dict was not allowed:
W0102: Dangerous default value {} as argument (dangerous-default-value).
was not aware that this is a persistent object:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html